src/EnvironmentCheckSuite.php

```diff
@@ -184,6 +199,15 @@ public static function register($names, $check, $title = null)
         if (!is_array($names)) {
             $names = array($names);
         }
+
+        /**
+         * Support for omitted namespaces - this will fall back to adding the SilverStripe\EnvironmentCheck\Checks
+         * namespace if one isn't provided. If you are using custom checks, make sure you include a namespace (or "\")
+         */
+        if (is_string($check) && strpos($check, '\\') === false) {
+            $check = __NAMESPACE__ . '\\Checks\\' . $check;
+        }
```

Comment:
Yep - I'm going to replace this with an Injector call instead as per @sminnee's suggestion